tests/test_gbdesAstrometricFit.py

```diff
@@ -539,6 +539,31 @@ def test_run(self):
                     np.testing.assert_array_less(absDiffX, 1e-7)
                     np.testing.assert_array_less(absDiffY, 1e-7)
 
+    def test_missingWcs(self):
+        """Test that task does not fail when the input WCS is None for one
+        extension and that the fit WCS for that extension returns a finite
+        result.
+        """
+        inputVisitSummary = self.inputVisitSummary.copy()
+        # Set one WCS to be None
+        testVisit = 0
+        testDetector = 20
+        inputVisitSummary[testVisit][testDetector].setWcs(None)
+
+        outputs = self.task.run(
+            self.inputCatalogRefs,
+            inputVisitSummary,
+            instrumentName=self.instrumentName,
+            refEpoch=self.refEpoch,
+            refObjectLoader=self.refObjectLoader,
+        )
+
+        # Check that the fit WCS for the extension with input WCS=None returns
+        # finite sky values.
```

Comment:
Should it return finite sky values, or correct sky values? Is the test dataset not large enough to get a reasonable fit in this case?

Reply:
The test data is simulated without the missing WCS, so we don't expect to be able to recover the same input in this test. It's possible to do a whole new simulation, but I don't think it's worthwhile in this case.